charts/operators/Chart.yaml

```diff
@@ -14,4 +14,4 @@ type: application
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 2.0.10-rc.5
+version: 2.0.11
```

Comment:
when I run the create_pelorus_operator.sh script, I get:

```
$ ./scripts/create_pelorus_operator.sh 
INFO: Source directory: /home/esauer/src/pelorus/scripts/../charts/pelorus/
INFO: Destination directory: /home/esauer/src/pelorus/scripts/../pelorus-operator/
INFO: Patches directory: /home/esauer/src/pelorus/scripts/pelorus-operator-patches/
INFO: operator-sdk version: "v1.28.1", commit: "b05f6a56a176a98b7d92c4d4b36076967e0d77f7", kubernetes version: "1.26.0", go version: "go1.19.8", GOOS: "linux", GOARCH: "amd64"
ERROR: Destination directory /home/esauer/src/pelorus/scripts/../pelorus-operator/ is not empty, can not continue.
```

Is the directory supposed to be empty?

Reply:
after running `make rc-release` you need to run `helm dep update charts/pelorus && rm -r charts/pelorus/charts/*.tgz` (this is going to added to the fix I am working on). This will update Chart.lock

Reply:
This is the reason I am working on the fix I think, the command should have updated it to `2.0.10-rc.6`

please run
```
source .venv/bin/activate
./scripts/create_release_pr.sh -v 2.0.10-rc.6
helm dep update charts/pelorus && rm -r charts/pelorus/charts/*.tgz
./scripts/create_pelorus_operator.sh -f -v 0.0.7-rc.6
```
This will update everything correctly (with the fix I am working, just running `make rc-release` will do the same)